src/protocol.h

```diff
@@ -225,6 +225,17 @@ extern const char* GETBLOCKTXN;
  * @since protocol version 70014 as described by BIP 152
  */
 extern const char* BLOCKTXN;
+/**
+ * getcfheaders requests compact filter headers for a range of blocks.
+ * Only available with service bit NODE_COMPACT_FILTERS as described by
+ * BIP 157 & 158.
+ */
+extern const char *GETCFHEADERS;
```

Comment:
style-nit in commit db20d456f0 (according to clang-format):

```suggestion
extern const char* GETCFHEADERS;
```

Reply:
Whoops. Missed this out in the rebase. Fixed.